lib/cooked_post_processor.rb

```diff
@@ -97,8 +96,8 @@ def extract_images_for_post
     # all image with a src attribute
     @doc.css("img[src]") -
     # minus, emojis
-    @doc.css("img.emoji") -
-    # minus, image inside oneboxes
+    @doc.css("img.emoji") +
+    # plus, image inside oneboxes
     oneboxed_images -
```

Comment:
You don't need to add the `oneboxed_images` because they're already found by `@doc.css("img[src]")`.